src/mame/drivers/amiga.cpp

```diff
@@ -100,7 +100,7 @@ class a1000_kbreset_device : public device_t
 
 	auto kbrst_cb() { return m_kbrst_cb.bind(); }
 
-	a1000_kbreset_device &set_delays(attotime detect, attotime stray, attotime output)
+	a1000_kbreset_device &set_delays(const attotime &detect, const attotime &stray, const attotime &output)
```

Comment:
Like the UML `parameter` the `attotime` is a literal class and can be passed by value on the stack .  It's not desirable to do this everywhere.

Reply:
I think this was the only case where `attotime` was not passed by reference - at least outside of `devices`, but I could be wrong.